dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -30,7 +30,15 @@ def perform(user_id:, script_level_id:)
 
     raise "lesson_s3_name not found for script_level_id: #{script_level.id}" if lesson_s3_name.blank?
 
-    puts "TODO: Implement the rest of this job"
+    channel_id = get_channel_id(user, script_level)
+    code, project_version = read_user_code(channel_id)
+    puts "code: #{code.inspect}"
+
+    rubric = Rubric.find_by!(lesson_id: script_level.lesson.id, level_id: script_level.level.id)
+
+    ai_evaluations = get_fake_openai_evaluations(rubric, understanding_s: 'Extensive Evidence')
```

Comment:
lol is this why I only got "Extensive Evidence" during the bug bash last week?

Reply:
haha, that WOULD explain it ! but no, this code was never on the adhoc. the adhoc was using pycall to make real calls to openai. this is just a tactic to break up the work and get all the non-python stuff merged while we figure out how to move from pycall to python service.